package.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "court-watch",
   "version": "1.0.0",
-  "description":
-    "A notifications, bookings and data gathering app for court observers at hearings where defendants are women.",
+  "description": "A notifications, bookings and data gathering app for court observers at hearings where defendants are women.",
```

Comment:
I didn't change anything on package.json (other than installing react-router-dom). This difference is probably due to the fact that we have different spacing settings in our editors :wink:

Reply:
yeah we should sort that out tomorrow!